packages/babel-core/src/config/config-chain.js

```diff
@@ -115,11 +116,34 @@ export function buildRootChain(
   );
   if (!programmaticChain) return null;
 
+  const { root = ".", babelrc = root, configFile } = opts;
+
+  let configFileObj = null;
+  if (configFile !== undefined) {
+    configFileObj = loadConfig(configFile, context.cwd, context.envName);
+  } else {
+    try {
+      configFileObj = loadConfig(
+        "./babel.config.js",
+        path.resolve(context.cwd, root),
+        context.envName,
+      );
+    } catch (e) {
+      if (e.code !== "MODULE_NOT_FOUND") throw e;
```

Comment:
Annoyingly, this may actually eat resolving errors that come from dependencies of the config file itself (e.g. if it tries to require a plugin to pass by value instead of passing it by name).

We probably need to do a `require.resolve` check first to ensure any exception we ignore is because the config isn't present.

Reply:
Good call. When I switch this up to be real production-ready code, I'll see what I can do.